docs/docs/Configuration/configuration-cli.md

```diff
@@ -33,7 +33,7 @@ python -m langflow [OPTIONS]
 
 ### langflow api-key
 
-Create an API key for the default superuser if the `LANGFLOW_AUTO_LOGIN` environment variable is set to `true`.
+To create API keys with the Langflow CLI, `AUTO_LOGIN` must be set to `FALSE` and you must be logged in as a superuser.
```

Comment:
Still unclear how I know I am a superuser. Is everyone a superuser by default because there's no rbac?